datalad/core/distributed/clone.py

```diff
@@ -585,15 +585,125 @@ def postclonecfg_ria(ds, props):
                    if s.get('annex-externaltype', None) == 'ora'
     ]
     if not ria_remotes:
-        lgr.debug("Found no RIA special remote")
+        # nothing autoenabled. Let's check origin's config for
+        # datalad.ora-remote.uuid as stored by create-sibling-ria and enable try
+        # enabling that one.
+        lgr.debug("Found no autoenabled ORA special remote. Trying to look it"
+                  "up in source config ...")
+
+        # First figure whether we cloned via SSH, HTTP or local path and then
+        # get that config file the same way:
+
+        config_content = None
+        scheme = props['source'].split(':')[0][4:]
```

Comment:
This assumes the `props['source']` is something like `ria+...://` which is true at this point (with the current code), but it could be relatively painful to decode this fact back from this line in a future RF. Maybe add a comment? Or strip `len('ria+')`?

Reply:
Agree. `lstrip("ria+")` might be most explicit.

Reply:
Careful with using `*strip()`: 'aaaaa'.lstrip('a')

Better stick to your pattern, but with `len()`

Reply:
Yeah - not concerned with "ria+ria+ria+" URLs actually, but still simply faster to with slicing.

Reply:
It would also strip `air+` or `ira+` or `ai`. OMG it removes any AI...

Reply:
> OMG it removes any AI...

Sounds like a feature ;-)